CHANGELOG.rst

```diff
@@ -3,9 +3,19 @@ Changelog
 
 0.11.3
 ------
-- Performance fix with PyPika for small fetch queries.
+* Test runner now has the ability to skip tests conditionally, based on the DB driver Capabilities:
+
+  .. code-block:: python3
+
+      @requireCapability(dialect='sqlite')
+      async def test_run_sqlite_only(self):
+          ...
+
+- Performance fix with PyPika for small fetch queries
 - Remove parameter hack now that PyPika support Parametrized queries
 - Fix typos in JSONField docstring
+- Added basic DB driver Capabilities
+- moo
```

Comment:
moo?

Reply:
:-( Don't write code when sleepy Nick

Reply:
There is more than one mistake there. The first & last point is the same.